ledger/src/blockstore.rs

```diff
@@ -755,7 +754,6 @@ impl Blockstore {
                             &mut erasure_metas,
                             &mut index_working_set,
                             &mut slot_meta_working_set,
-                            &mut write_batch,
                             &mut just_inserted_data_shreds,
```

Comment:
Do you handle purging shreds?

Reply:
I believe so. It removes those slot directories

Reply:
Ah, I see it. The diff was hiding the purge function changes.